themes/bootstrap3/templates/ContentBlock/FacetList.phtml

```diff
@@ -11,7 +11,7 @@
       <?php if ($isHierarchy = in_array($field, $hierarchicalFacets ?? [])):
           $this->headScript()->appendFile('vendor/jsTree/jstree.min.js');
           $this->headScript()->appendFile('facets.js');
-          $sort = $hierarchicalFacetSortOptions[$field] ?? '';
+          $sort = $hierarchicalFacetSortOptions[$field] ?? $hierarchicalFacetSortOptions['*'] ?? 'count';
```

Comment:
Was it a bug that the $sort variable was never used before?

Reply:
Yes, I think so...